llvm/test/DebugInfo/verify-dwarf-no-operands.ll

```diff
@@ -0,0 +1,30 @@
+; RUN: llc --filetype=obj %s -o -
```

Comment:
This will not work in a build without a backend. You should probably move this with the other verifier tests in test/Verifier and just run llvm-as on it.

Reply:
Can you also comment which line is the interesting one?